app/frontend/src/search/client.js

```diff
@@ -8,7 +8,10 @@ import { getRadius } from './ui/input/radius';
 
 // This is the public API key which can be safely used in your frontend code.
 // This key is usable for search queries and list the indices you've got access to.
-export const search = algoliasearch('QM2YE0HRBW', '4082ba44346a92023eac1f794d739dd1');
+export const search = algoliasearch(
+  document.getElementById('new_').getAttribute('data-app'),
```

Comment:
The id being `new_` is a bug. (Possibly I think due the form builder not knowing that name of what it's been passed), so we should probably sort this

Reply:
i havent fixed the source problem but this code is no longer dependant on the erroneous id